CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to Thermo
+
+## Overview
+Welcome to the Thermo Repo! Here, you will find everything you need to start contributing to the Thermo Library. All you need is a GitHub account and the ability to explain how to solve a problem. If we accept your submission, our editors will review your content and direct you to make necessary changes before merging your content with the Library.
+
+## What you need
+* Expertise in your topic
+* A tested, verified procedure or procedures for a topic related to Thermo
+* A GitHub account and a fork of the Thermo repo.
+
+## Table of contents
+* [Crucial criteria](#crucial-criteria)
+* [Payments](#payments)
+* [Submission process](#submission-process)
+* [Style guide](#style-guide)
+* [Template](#template)
+* [Using images](#using-images)
+
+## Crucial criteria
+* **Accuracy:** Your procedures are straightforward, correct, and tested. For each action, include the “why” as well as the “what.” Explain the purpose behind each action.
+* **Completeness:** Your guide produces a complete, functional configuration. When appropriate, your guide also gives readers of what to do next and provides all critical elements of security and other best practices.
```

Comment:
Check the wording of this sentence.

Reply:
changed "gives readers of what do to next" to "gives readers ideas about what to do next..."